### PR TITLE
JENKINS-47749 Add editor test for creating a parallelized pipeline

### PR DESCRIPTION
**Pull request by kshultzCB**
# Description

See [JENKINS-47749](https://issues.jenkins-ci.org/browse/JENKINS-47749). This PR is for the addition of a Pipeline Editor test which creates a Parallel pipeline, in support of the parallel syntax introduced with [Declarative 1.2](https://wiki.jenkins.io/display/JENKINS/Pipeline+Model+Definition+Plugin). This PR also includes some small README edits.

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests: this is an acceptabce test addition. Unit and acceptance tests not applicable
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

# To run:

1. Follow the instructions provided in [the acceptance tests' README](https://github.com/kshultzCB/blueocean-plugin/tree/add-parallel-editor-test/acceptance-tests) for setting up the "live" acceptance tests
2. Invoke `mvn test -Dprofile=live -Dtest=GithubEditorTest#testEditorParallel`

@kzantow , @imeredith , a review would be most appreciated.

### CODE REVIEW

**Comment by michaelneale** [comment]
very nice @kshultzCB !!

**Comment by michaelneale** [comment]
ping @imeredith @cliffmeyers look ok? LGTM

File: acceptance-tests/README.md [hunk 1]
```diff
@@ -271,8 +270,8 @@ Any new Page Object classes need to be bound in `ATHModule#configure` to be able
 
 #### Pipeline Helper
 
-These are a series or helpers to deal with pipelines. The started life to deal with pieplines being in folders. It offes a way 
-to give PageOjects more contextual information about what they are operating on without having to be explicit about it in every method call.
+These are a series of helpers to deal with pipelines. They started life to deal with pipelines being in folders. It offes a way 
+to give PageObjects more contextual information about what they are operating on without having to be explicit about it in every method call.
```
Comment (sophistifunk) [inline acceptance-tests/README.md]: These are all great

**Comment by michaelneale** [comment]
@kshultzCB +100 hacker points

**Comment by i386** [comment]
Nice job @kshultzCB 💯

**Pull request merged**